build.gradle

```diff
@@ -60,28 +51,22 @@ java {
     withSourcesJar()
 }
 
-clean {
-    delete sharedDir
-}
-
 dependencies {
-    api platform('com.exactpro.th2:bom:3.0.0')
+    api platform('com.exactpro.th2:bom:4.5.0')
 
-    implementation 'com.exactpro.th2:common:3.22.0'
-    implementation 'com.exactpro.th2:grpc-conn:0.0.1'
-    implementation 'com.exactpro.th2:conn-ws-client:0.2.3'
-    implementation 'com.exactpro.th2:conn-http-client:0.1.1'
+    implementation 'com.exactpro.th2:common:5.4.1-dev'
+    implementation 'com.exactpro.th2:common-utils:2.2.0-dev'
+    implementation 'com.exactpro.th2:grpc-conn:0.1.0-TS-1663-+'
+    implementation 'com.exactpro.th2:conn-ws-client:0.4.0-TS-1633-+'
+    implementation 'com.exactpro.th2:conn-http-client:2.1.0-dev'
 
-    implementation 'org.slf4j:slf4j-log4j12:1.7.32'
-    implementation 'org.slf4j:slf4j-api:1.7.32'
+    implementation 'org.slf4j:slf4j-api'
 
-    implementation group: 'org.jetbrains.kotlin', name: 'kotlin-stdlib-jdk8'
-    implementation group: 'org.jetbrains.kotlin', name: 'kotlin-reflect'
-    implementation group: 'io.github.microutils', name: 'kotlin-logging', version: '1.7.9'
-    implementation group: 'com.fasterxml.jackson.module', name: 'jackson-module-kotlin', version: '2.11.2'
-    implementation group: 'com.athaydes.rawhttp', name:'rawhttp-core', version: '2.4.1'
+    implementation group: 'io.github.microutils', name: 'kotlin-logging', version: '3.0.5'
+    implementation group: 'com.fasterxml.jackson.module', name: 'jackson-module-kotlin', version: '2.15.2'
+    implementation group: 'com.athaydes.rawhttp', name:'rawhttp-core', version: '2.6.0'
 
-    testImplementation group: 'org.jetbrains.kotlin', name: 'kotlin-test-junit5'
+    testImplementation group: 'org.jetbrains.kotlin', name: 'kotlin-test-junit5', version: kotlin_version
```

Comment:
We don't have to specify version for kotlin dependencies because kotiln plugin does it by itself